src/kirby/components/list/list.component.html

```diff
@@ -11,7 +11,7 @@
         >
         </ng-container>
       </li>
-      <li class="row" *ngFor="let item of section.items" (click)="onItemClick(item)" >
+      <li class="row" [ngClass]="{'clickable': isSelectable}" *ngFor="let item of section.items" (click)="onItemClick(item)" tabindex="0">
```

Comment:
Maybe we should use the [Aria button role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) on the `li` tags that are selectable.

Reply:
I belive I changed that and pushed it with a few other changes